templates/data/mysql.html

```diff
@@ -1,477 +1,533 @@
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/bf2f1cc2-maas-logo false.png",
+                        alt="MAAS",
+                        width="250",
+                        height="312",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+          </div>
         </div>
       </div>
-      <hr class="p-rule">
-      <div class="row">
-        <div class="col-3 col-3 col-medium-2">
-          <h3 class="p-heading--5">FinOps and banking</h3>
-        </div>
-        <div class="col-3 col-medium-2">
-          <p>MySQL provides the transactional <a href="https://en.wikipedia.org/wiki/ACID">ACID</a> guarantees and the mature replication capabilities that finance systems need.</p>
-        </div>
-        <div class="col-3 col-medium-2">
-          {{ 
-            image (
-            url="https://assets.ubuntu.com/v1/f8b601e5-FinOps and banking@3x.png",
-            alt="",
-            width="901",
-            height="600",
-            hi_def=True,
-            loading="lazy"
-            ) | safe
-          }}
-        </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Easy to deploy and manage</h2>
       </div>
-      <hr class="p-rule">
-      <div class="row">
-        <div class="col-3 col-medium-2">
-          <h3 class="p-heading--5">Text search use cases</h3>
-        </div>
-        <div class="col-3 col-medium-2">
-          <p>MySQL also provides basic <a href="https://dev.mysql.com/doc/refman/8.0/en/fulltext-search.html">full text search</a> capabilities allowing you to accommodate such use cases without the need for duplicating your data or compromising on its consistency.</p>
+      <div class="col">
+        <div class="p-section--shallow">
+          <div class="p-image-container is-highlighted u-hide--small">
+            {{ image(url="https://assets.ubuntu.com/v1/3815a62c-easy-to-deploy-manage.png",
+                        alt="",
+                        width="1800",
+                        height="1015",
+                        hi_def=True,
+                        attrs={"class": "p-image-container__image"},
+                        loading="lazy") | safe
+            }}
+          </div>
         </div>
-        <div class="col-3 col-medium-2">
-          {{ 
-            image (
-            url="https://assets.ubuntu.com/v1/868a5f26-Text search use cases@3x.png",
-            alt="",
-            width="901",
-            height="600",
-            hi_def=True,
-            loading="lazy"
-            ) | safe
-          }}
+        <p>Thanks to Charmed MySQL, your deployments will be:</p>
+        <hr class="p-rule--muted" />
```

Comment:
Pls use is-muted class everywhere, your spacing is off because of the p-rule--muted